thingsnotion.py

```diff
@@ -101,6 +118,46 @@ def create_callout_block(title='', children=[]):
         },
     }
 
+def create_toggle_block(title='', children=[]):
+    return {
+        "object": 'block',
+        "type": 'toggle',
+        "has_children": True,
+        "toggle": {
+            "rich_text": [{
+                "type": "text",
+                "text": {
+                    "content": title,
+                },
+            }],
+            "color": "default",
+            "children": children
+        },
+    }
+
+def create_block(type: BlockTypes, title='', children=[]):
```

Comment:
can we comment out if we don't want to use ATM if it isn't working -- and add todo to use this to unify all